src/submit-modal/success-content.js

```diff
@@ -21,15 +21,15 @@ import styles from './success-content.module.css'
 const importTypeConfig = {
     imported: {
         style: 'success',
-        text: i18n.t('Imported'),
+        text: i18n.t('imported'),
```

Comment:
the change here is not related to the main purpose of the ticket, but I noticed incorrect translations when adding/checking translations for this ticket. Lowercasing the translation did not appear to work here, so instead the translation strings in lower-case form are added directly